README.md

```diff
@@ -13,9 +13,9 @@
     - [functions.sh](#functions.sh)
     - [get_options.sh](#get_options.sh)
     - [autosetup.sh](#autosetup.sh)
-    - [setup.sh](setup.sh)
+    - [setup.sh](#setup.sh)
     - [install.sh](#install.sh)
```

Comment:
All these '#' Anchor should be like '#autosetupsh' instead of '#autosetup.sh'.
Reason: At the moment clicking the link in the rendered README.md does not jump to the correct point.
e.g: 'https://github.com/virtapi/installimage/blob/31f05c92d71f97c624646d1e7f2b93a28a8a1788/README.md#config.sh'
does not work while
'https://github.com/virtapi/installimage/blob/31f05c92d71f97c624646d1e7f2b93a28a8a1788/README.md#configsh'
does.